client/src/components/PageExplorer/PageExplorer.scss

```diff
@@ -61,10 +61,14 @@ $explorer-header-horizontal-padding: 10px;
 .c-page-explorer__header__title__inner {
   @apply w-flex;
   padding: 1em $explorer-header-horizontal-padding;
+  padding-bottom: 1rem;
```

Comment:
```suggestion
```

This is not really needed as it overrides the existing padding - it is best to resolve the size of something from the outside in if possible (especially when trying to match with an outside element).